[New Product] Silverstripe CMS

dcentrica opened the pull request:
Added new product entry for: Silverstripe CMS (https://silverstripe.org)

phptek committed:
NEW PRODUCT: Silverstripe CMS

captn3m0 commented on products/silverstripe.md:
```diff
@@ -0,0 +1,66 @@
+releaseImage: https://www.silverstripe.org/assets/Uploads/_resampled/ResizedImageWzYwMCwzOTdd/CMS-5-Support-Timeline-May-2023.png 
+releasePolicyLink: https://www.silverstripe.org/software/roadmap/
+changelogTemplate: https://github.com/silverstripe/silverstripe-cms/releases/__LATEST__
+activeSupportColumn: true
```
Match the upstream terminology better:

```suggestion
activeSupportColumn: ActiveDevelopment
eolColumn: Security Support
```

captn3m0 commented on products/silverstripe.md:
```diff
@@ -0,0 +1,66 @@
+    eol: 2028-04-01
+    latest: "5.0.4"
+    latestReleaseDate: 2023-05-31
+    releaseDate: 2023-05-01
```
Since the dates for 5.0 are currently undecided, and we're marking this as `5.0` (not `5`), lets just mark it as "currently supported" using `true/false`. Once the dates are announced, we can adjust accordingly.

```suggestion
-   releaseCycle: "5.0"
    support: true
    eol: false
    latest: "5.0.4"
    latestReleaseDate: 2023-05-31
    releaseDate: 2023-05-01
```

captn3m0 commented on products/silverstripe.md:
```diff
@@ -0,0 +1,66 @@
+    releaseDate: 2023-05-01
+
+-   releaseCycle: "4.13"
+    support: 2025-04-01
```
Active Development ends in April 2024 for 4.13

```suggestion
    support: 2024-04-01
```

captn3m0 commented on products/silverstripe.md:
```diff
@@ -0,0 +1,66 @@
+    releaseDate: 2023-04-26
+
+-   releaseCycle: "4.12"
+    support: 2023-10-26
```
```suggestion
    support: 2023-04-26
```

captn3m0 commented on products/silverstripe.md:
```diff
@@ -0,0 +1,66 @@
+    releaseDate: 2022-11-30
+
+-   releaseCycle: "4.11"
+    support: 2023-05-19
```
```suggestion
    support: 2022-12-19
```

captn3m0 commented on products/silverstripe.md:
```diff
@@ -0,0 +1,66 @@
+    releaseDate: 2022-06-28
+
+-   releaseCycle: "4.10"
+    support: 2022-12-31
```
```suggestion
    support: 2022-06-30
```

captn3m0 reviewed (changes requested):
Minor suggestions to better align with the upstream.

captn3m0 commented on products/silverstripe.md:
```diff
@@ -0,0 +1,66 @@
+
+> [Silverstripe](https://www.silverstripe.org/) is a free and open-source content management CMS and Framework written in PHP and distributed under the BSD License.
+
+For major releases that are in "Active development" or "Full support", minor versions are released approximately every three months, and their End-of-Life (EOL) is announced at least six months in advance. The latest minor release is supported as long as its underlying major release.
```
Since we're not mentioning the v5 EOL date in the table, lets include it here:

```suggestion
For major releases that are in "Active development" or "Full support", minor versions are released approximately every three months, and their End-of-Life (EOL) is announced at least six months in advance. The latest minor release is supported as long as its underlying major release.

Silverstripe 5 will receive security fixes till April 2027.
```

dcentrica committed:
Update products/silverstripe.md

Minor changes to better reflect upstream status

Co-authored-by: Nemo <me@captnemo.in>

dcentrica committed:
Update products/silverstripe.md

Substitute an actual date for values which denote "currently supported" for 5.0.

Co-authored-by: Nemo <me@captnemo.in>

dcentrica committed:
Update products/silverstripe.md

Fixes stupid mistake, reading from the wrong column.

Co-authored-by: Nemo <me@captnemo.in>

dcentrica committed:
Update products/silverstripe.md

Fixes stupid mistake, reading from the wrong column.

Co-authored-by: Nemo <me@captnemo.in>

dcentrica committed:
Update products/silverstripe.md

Fixes stupid mistake, reading from the wrong column.

Co-authored-by: Nemo <me@captnemo.in>

dcentrica committed:
Update products/silverstripe.md

Fixes stupid mistake, reading from the wrong column.

Co-authored-by: Nemo <me@captnemo.in>

dcentrica committed:
Update products/silverstripe.md

Accept suggested wording tweak.

Co-authored-by: Nemo <me@captnemo.in>

captn3m0 committed:
Update silverstripe.md

captn3m0 reviewed (approved):
LGTM.
